rust/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:experimental
 
-FROM rust:1.56 as builder
+FROM rust:1.60 as builder
```

Comment:
I assume you have tested that this does not break anything

Reply:
Let me run now that I have that setup, but I would be genuinely amazed if a Rust version bump broke anything.